doc/API-Entities.md

```diff
@@ -908,6 +908,13 @@ Identical to [the Twitter Media Object](https://developer.twitter.com/en/docs/tw
 <td>Resource ID (32 hex chars)</td>
 </tr>
 
+<tr>
+<td><code>media-id</code></td>
+<td>String (Integer) </td>
+<td>ID used for attaching to a Mastodon Post Status</td>
```

Comment:
Attaching media? Files?

Reply:
I guess generally it would be other media attachments but right now it's only tested with photos/images

Reply:
Got it, can you please mention this in the notes?

Reply:
I'll see what I can do on the rephrasing because I'm not following what is missing here.

Reply:
```suggestion
<td>ID used for attaching images to a Mastodon Post Status</td>
```